### PR TITLE
tlx 0.5.2 (new formula)

### PR DESCRIPTION
**Pull request by fabratu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/tlx.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  sha256 "5e67d3042a390dbb831b6d46437e3c7fadf738bff362aa7376b210b10ecd532d"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = std_cmake_args + [".."]
+    mkdir "build" do
+      system "cmake", ".", *args
+      system "make", "install"
+    end
```
Comment (chenrui333) [inline Formula/tlx.rb]: Maybe simple as this?
```suggestion
    system "cmake", "."
    system "make", "install"
```

**Comment by SMillerDev** [comment]
```
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:23 (message):
  In-source builds are not allowed.


Defaulting CMAKE_BUILD_TYPE to Debug
-- Performing Test TLX_CXX_HAS_CXX17
-- Performing Test TLX_CXX_HAS_CXX17 - Success
-- TLX CMAKE_CXX_FLAGS: -Wshadow -Wold-style-cast -std=c++17 -g -W -Wall -Wextra -fPIC  -Wdeprecated
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Configuring incomplete, errors occurred!
See also "/tmp/tlx-20200308-58321-go748a/tlx-0.5.20191212/CMakeFiles/CMakeOutput.log".
```

**Comment by fabratu** [comment]
That is why the build-folder was intended in the first version. Besides being shorter/easier, is there a reason not to do this inside a formula? (Can't remember the example, but saw this creation of build-folders for different other formulas as well.)

**Comment by fabratu** [comment]
For now the state of the formula is back to its initial state. If it is needed to change it to an in-source build, I will see if a patch/update of the original package is possible.

**Comment by SMillerDev** [comment]
Thanks @fabratu ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**